package.json

```diff
@@ -3,11 +3,22 @@
   "private": true,
   "dependencies": {
     "@rails/ujs": "^6.1.3-1",
-    "@rails/webpacker": "^5.2.1",
-    "turbolinks": "^5.2.0"
+    "@rails/webpacker": "^6.0.0-beta.5",
```

Comment:
This now needs to be `6.0.0-beta.7` (without the `^`) to be in sync with the Gem version.